Corrected programming language for article

Youssef1313 opened the pull request:
* Fixes #13331.
* Corrected `dev_langs` meta data.
* Followed coding conventions (using var instead of data type when data type is clear from right hand side)

Youssef1313 committed:
Corrected programming language for article

Youssef1313 commented:
I'm going to push more commits to this PR. Don't merge now.

BillWagner commented:
@Youssef1313 

> I'm going to review some related articles and may push more commits to this PR. Don't merge now.

I changed the title to add “WIP” for “Work in progress” to the title. We recognize that tag for any PR that’s still being worked on and isn’t ready to merge. Just remove that tag when you’re ready for final review and merge.

Youssef1313 committed:
Update broken link.

Youssef1313 commented:
> 
> 
> @Youssef1313
> 
> > I'm going to review some related articles and may push more commits to this PR. Don't merge now.
> 
> I changed the title to add “WIP” for “Work in progress” to the title. We recognize that tag for any PR that’s still being worked on and isn’t ready to merge. Just remove that tag when you’re ready for final review and merge.

Thanks for the info about "WIP".
I've updated a broken link, but not sure if the new link is good replacement or not.

BillWagner commented on docs/framework/wcf/feature-details/transport-security-with-basic-authentication.md:
```diff
@@ -7,7 +7,7 @@ dev_langs:
 ---
 # Transport Security with Basic Authentication
-The following illustration shows a Windows Communication Foundation (WCF) service and client. The server needs a valid X.509 certificate that can be used for Secure Sockets Layer (SSL), and the clients must trust the server’s certificate. Further, the Web service already has an SSL implementation that can be used. For more information about enabling basic authentication on Internet Information Se ...
+The following illustration shows a Windows Communication Foundation (WCF) service and client. The server needs a valid X.509 certificate that can be used for Secure Sockets Layer (SSL), and the clients must trust the server’s certificate. Further, the Web service already has an SSL implementation that can be used. For more information about enabling basic authentication on Internet Information Se ...
```
We remove the locales from links so that they resolve to the users' culture / region:

```suggestion
The following illustration shows a Windows Communication Foundation (WCF) service and client. The server needs a valid X.509 certificate that can be used for Secure Sockets Layer (SSL), and the clients must trust the server’s certificate. Further, the Web service already has an SSL implementation that can be used. For more information about enabling basic authentication on Internet Information Services (IIS), see <https://docs.microsoft.com/iis/configuration/system.webserver/security/authentication/basicauthentication>.  
```

BillWagner commented:
Thanks for these changes @Youssef1313 

I had one update, and once you make that change, I'll :shipit:.

Youssef1313 committed:
Update docs/framework/wcf/feature-details/transport-security-with-basic-authentication.md

Co-Authored-By: Bill Wagner <wiwagn@microsoft.com>

Youssef1313 commented:
Thanks for the update, @BillWagner. I've committed it.
